tests/e2e/specs/auth-setup-verification.test.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-env node */
+/* global page */
+
+/**
+ * WordPress dependencies
+ */
+import { activatePlugin, createURL, visitAdminPage } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { deactivateAllOtherPlugins, resetSiteKit } from '../utils';
+
+const oauthClientConfig = JSON.stringify( {
+	'web': {
+		'client_id': 'test-client-id',
+		'client_secret': 'test-client-secret',
+		'project_id': 'test-project-id',
+		'auth_uri': 'https://accounts.google.com/o/oauth2/auth',
+		'token_uri': 'https://accounts.google.com/o/oauth2/token',
+		'auth_provider_x509_cert_url': 'https://www.googleapis.com/oauth2/v1/certs'
+	}
+} );
+
+/**
+ * @link https://stackoverflow.com/a/46012210/1037938
+ */
+async function simulatePastingClientConfiguration() {
```

Comment:
Curious, why exactly does regular `page.type()` not work? Is it a limitation in the setup component?

Reply:
`page.type()` works, but it's terribly slow in the interactive mode, and no user is ever going to type that in, so it seemed more "real" to simulate pasting it in all at once.

Reply:
Suggestion: move this to a new `pasteText( inputEl, text )` utility function and use it in `tests/e2e/specs/auth-flow.test.js` too.

Reply:
I thought about that which is a good idea, it would need a little reworking though as the implementation is Element class-specific. We should be able to derive that from the element itself I would think though. I think that would be a great utility to have though, and I was a bit surprised that core didn't have something for this purpose, considering the paste handling present in Gutenberg.

Reply:
Suggestion applied and working great! 😄

Reply:
Nice!